x-pack/platform/plugins/shared/alerting/server/application/maintenance_window/methods/create/create_maintenance_window.ts

```diff
@@ -63,13 +63,17 @@ export async function createMaintenanceWindow(
     );
   }
 
-  if (scopedQueryWithGeneratedValue) {
-    if (data.categoryIds?.length !== 1) {
-      throw Boom.badRequest(
-        `Error validating create maintenance window data - scoped query must be accompanied by 1 category ID`
-      );
-    }
-  }
+  // This validation was removed so that the new POST maintenance window API can be created.
+  // The categoryIDs field will be completely removed in the future.
+  // https://github.com/elastic/kibana/issues/197530
+  //
+  // if (scopedQueryWithGeneratedValue) {
```

Comment:
This was discussed offline. I opted to leave it commented out for clarity but can also remove it.